test/dialect/postgresql/test_types.py

```diff
@@ -3673,6 +3680,490 @@ class DateTimeTZRangeRoundTripTest(_DateTimeTZRangeTests, _RangeTypeRoundTrip):
     pass
 
 
+class _MultiRangeTypeCompilation(AssertsCompiledSQL, fixtures.TestBase):
+    __requires__ = "range_types", "psycopg_only_compatibility"
```

Comment:
**mike bayer** (zzzeek) wrote:

these are compile tests, so we don't need psycopg_only_compat here, this is just running the strings

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3728

Reply:
removed.